app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -1,6 +1,12 @@
-attributes :content_view_id, :content_view_name
-attributes :lifecycle_environment_id, :lifecycle_environment_name
-attributes :uuid
+child :content_view => :content_view do
+  attributes :id, :name
+end
+
+child :lifecycle_environment => :lifecycle_environment do
+  attributes :id, :name
+end
```

Comment:
I don't really understand the point of having the other method, Does something in the UI make it easier?